docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -402,11 +402,20 @@ WINDOWS: If your Windows log rotation system shows errors because it can’t rot
 
 #### `close.on_state_change.removed` [filebeat-input-filestream-close-removed]
 
-When this option is enabled, Filebeat closes the harvester when a file is removed. Normally a file should only be removed after it’s inactive for the duration specified by `close.on_state_change.inactive`. However, if a file is removed early and you don’t enable `close.on_state_change.removed`, Filebeat keeps the file open to make sure the harvester has completed. If this setting results in files that are not completely read because they are removed from disk too early, disable this option.
-
-This option is enabled by default. If you disable this option, you must also disable `clean_removed`.
-
-WINDOWS: If your Windows log rotation system shows errors because it can’t rotate files, make sure this option is enabled.
+When this option is enabled, Filebeat closes the harvester when a file
+is removed. Normally a file should only be removed after it’s inactive
+for the duration specified by
+`close.on_state_change.inactive`. However, if a file is removed early
+and you don’t enable `close.on_state_change.removed`, Filebeat keeps
+the file open to make sure the harvester has completed. If this
+setting results in files that are not completely read because they are
+removed from disk too early, disable this option.
+
+This option is disabled by default on Linux and MacOS, on Windows it
+is enabled by default.
+
+WINDOWS: If your Windows log rotation system shows errors because it
+can’t rotate files, make sure this option is enabled.
```

Comment:
I think this lacks visibility

<img width="762" alt="Screenshot 2025-04-22 at 10 00 19" src="https://github.com/user-attachments/assets/a6fce5ea-afeb-42f0-b05a-6e055f0aba00" />

We should replace these `WINDOWS:` blocks with something like `::::{important}` in both `close.on_state_change.removed` and `close.on_state_change.renamed`.